app/forms/batch_edit_form.rb

```diff
@@ -57,6 +57,9 @@ def self.model_attributes(params)
     params[:rights] = Array(params[:rights]) if params[:rights].present?
     clean_params = super #hydra-editor/app/forms/hydra_editor/form.rb:54
     clean_params = encode_external_id(params, clean_params)
+    # These are removed due to a bug released in sufia 7.3:
+    # https://github.com/projecthydra-labs/hyrax/issues/652
+    clean_params['permissions_attributes'] = params['permissions_attributes'] if params['permissions_attributes']
     clean_params
```

Comment:
Here is a [debug gist](https://gist.github.com/hackmastera/f169c5b2e36373bbd0dd226fe0d9b1c0). I don't see any permissions checking, really, except the buggy implementation at `sufia-7.3.0/app/forms/sufia/forms/work_form.rb @ line 50` which is what is breaking this for batch edit (a batch edit form doesn't have an admin set).

I think permissions checking stuff usually happens during persistence; I'm seeing evidence of it in the actor stack. Makes me a bit confused why the bug was introduced this way to begin with. I'll ask about it on the hyrax ticket (https://github.com/projecthydra-labs/hyrax/issues/652)

I can think of a couple other ways to get around this, but they more or less do the same thing, which is circumnavigate the [upstream sanitize_params](https://gist.github.com/hackmastera/f169c5b2e36373bbd0dd226fe0d9b1c0#file-gistfile1-txt-L39) which is causing the trouble, and none of them are prettier.